docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Categorical/OneHotEncoding.cs

```diff
@@ -2,55 +2,65 @@
 using System.Collections.Generic;
 using Microsoft.ML;
 using Microsoft.ML.Data;
-using static Microsoft.ML.Transforms.OneHotEncodingEstimator;
+using Microsoft.ML.Transforms;
 
-namespace Samples.Dynamic
+namespace Samples.Dynamic.Transforms.Categorical
 {
     public static class OneHotEncoding
     {
         public static void Example()
         {
-            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
-            // as well as the source of randomness.
+            // Create a new ML context, for ML.NET operations. It can be used for
+            // exception tracking and logging, as well as the source of randomness.
             var mlContext = new MLContext();
 
             // Get a small dataset as an IEnumerable.
-            var samples = new List<DataPoint>()
+            var samples = new List<DataPoint>
             {
-                new DataPoint(){ Education = "0-5yrs" },
-                new DataPoint(){ Education = "0-5yrs" },
-                new DataPoint(){ Education = "6-11yrs" },
-                new DataPoint(){ Education = "6-11yrs" },
-                new DataPoint(){ Education = "11-15yrs" },
+                new DataPoint {Education = "0-5yrs"},
+                new DataPoint {Education = "0-5yrs"},
+                new DataPoint {Education = "6-11yrs"},
+                new DataPoint {Education = "6-11yrs"},
+                new DataPoint {Education = "11-15yrs"}
             };
 
             // Convert training data to IDataView.
-            var data = mlContext.Data.LoadFromEnumerable(samples);
+            IDataView data = mlContext.Data.LoadFromEnumerable(samples);
 
             // A pipeline for one hot encoding the Education column.
-            var pipeline = mlContext.Transforms.Categorical.OneHotEncoding("EducationOneHotEncoded", "Education");
+            OneHotEncodingEstimator pipeline =
+                mlContext.Transforms.Categorical.OneHotEncoding(
+                    "EducationOneHotEncoded","Education");
 
             // Fit and transform the data.
-            var oneHotEncodedData = pipeline.Fit(data).Transform(data);
+            IDataView oneHotEncodedData = pipeline.Fit(data).Transform(data);
```

Comment:
>IDataView [](start = 12, length = 9)

please coordinate with Artidoro on 'var' changes:
[https://github.com/dotnet/machinelearning/pull/3595#discussion_r278765890](https://github.com/dotnet/machinelearning/pull/3595#discussion_r278765890)

Reply:
we discussed and decided to keep everything as var except for IDataView.

---
In reply to: [278766172](https://github.com/dotnet/machinelearning/pull/3588#discussion_r278766172) [](ancestors = 278766172)